AsyncDisplayKit/ASCollectionView.h

```diff
@@ -171,6 +171,13 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)relayoutItems;
 
 /**
+ * Triggers a recalulation of leadingScreensForBatching
+ * @discussion During various layout changes that trigger content size changes, batch fetching needs to recalulate it's number of screens left to scroll before the delegate -collectionView:beginBatchFetchingWithContext: is called. Call when a layout transition is complete, or it will have no effect.
+ *
+ */
+- (void)invalidateBatchFetchMeasurements;
```

Comment:
@nickvelloff Thanks!  I see the need for this, but I do _not_ see a need for it to be a public API.

Could you explain?  Are you confident it can't be called internally, by something like relayoutItems or similar?